File: php/raindrops/Raindrops.php
<?php

declare(strict_types=1);

function raindrops(int $number): string
{
    $result = "";
    $isDivisable = false;
    if ($number % 3 === 0) {
        $result .= "Pling";
        $isDivisable = true;
    }
    if ($number % 5 === 0) {
        $result .= "Plang";
        $isDivisable = true;
    }
    if ($number % 7 === 0) {
        $result .= "Plong";
        $isDivisable = true;
    }

    if ($isDivisable === false) {
        $result = (string) $number;
    }

    return $result;
}

File: fsharp/raindrops/Raindrops.fs
module Raindrops

let convert (number: int): string =
    match number % 3 = 0, number % 5 = 0, number % 7 = 0 with
        | true, false, false -> "Pling"
        | false, true, false -> "Plang"
        | false, false, true -> "Plong"
        | true, true, false -> "PlingPlang"
        | true, true, true -> "PlingPlangPlong"
        | true, false, true -> "PlingPlong"
        | false, true, true -> "PlangPlong"
        | false, false, false -> string number
